<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Aggregator</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/resources/images/aggregator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="MainSplitter_" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Tags:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="TagsLine_" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="Feeds_" >
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="itemsExpandable" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ChannelDashboard_" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>140</height>
           </size>
          </property>
          <property name="title" >
           <string>Channel dashboard</string>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" colspan="2" >
            <widget class="QLabel" name="ChannelImage_" >
             <property name="text" >
              <string>&lt;channel image></string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Tags:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLineEdit" name="ChannelTags_" />
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QLabel" name="ChannelDescription_" >
             <property name="text" >
              <string>&lt;description></string>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Author:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" >
            <widget class="QLabel" name="ChannelAuthor_" >
             <property name="text" >
              <string>&lt;author></string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" >
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Language:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" >
            <widget class="QLabel" name="ChannelLanguage_" >
             <property name="text" >
              <string>&lt;language></string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="ChannelLink_" >
             <property name="text" >
              <string>&lt;url></string>
             </property>
             <property name="openExternalLinks" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_10" >
             <property name="text" >
              <string>Link:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Fixed string search:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="FixedStringSearch_" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Wildcard search:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="WildcardSearch_" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Regexp search:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="RegexpSearch_" />
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="CaseSensitiveSearch_" >
            <property name="text" >
             <string>Case sensitive</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeView" name="Items_" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <property name="itemsExpandable" >
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QTextBrowser" name="ItemView_" >
           <property name="autoFormatting" >
            <set>QTextEdit::AutoAll</set>
           </property>
           <property name="openExternalLinks" >
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions" >
    <property name="title" >
     <string>Actions</string>
    </property>
    <addaction name="ActionAddFeed_" />
    <addaction name="ActionRemoveFeed_" />
    <addaction name="ActionUpdateFeeds_" />
    <addaction name="separator" />
    <addaction name="ActionPreferences_" />
   </widget>
   <addaction name="menuActions" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionAddFeed_" />
   <addaction name="ActionRemoveFeed_" />
   <addaction name="ActionUpdateFeeds_" />
   <addaction name="ActionUpdateSelectedFeed_" />
   <addaction name="separator" />
   <addaction name="ActionPreferences_" />
  </widget>
  <action name="ActionAddFeed_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/rss-feed.png</iconset>
   </property>
   <property name="text" >
    <string>Add feed...</string>
   </property>
  </action>
  <action name="ActionPreferences_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/preferences.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="ActionUpdateFeeds_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/updateallfeeds.png</iconset>
   </property>
   <property name="text" >
    <string>Update feeds</string>
   </property>
  </action>
  <action name="ActionRemoveFeed_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/removefeed.png</iconset>
   </property>
   <property name="text" >
    <string>Remove feed</string>
   </property>
  </action>
  <action name="ActionMarkItemAsUnread_" >
   <property name="text" >
    <string>Mark item as unread</string>
   </property>
  </action>
  <action name="ActionMarkChannelAsRead_" >
   <property name="text" >
    <string>Mark channel as read</string>
   </property>
  </action>
  <action name="ActionMarkChannelAsUnread_" >
   <property name="text" >
    <string>Mark channel as unread</string>
   </property>
  </action>
  <action name="ActionUpdateSelectedFeed_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/updateselectedfeed.png</iconset>
   </property>
   <property name="text" >
    <string>Update selected feed</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
